deploy/setup.sh

```diff
@@ -0,0 +1,280 @@
+   echo "mongodb-enterprise-server hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-shell hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-mongos hold" | sudo dpkg --set-selections
+   echo "mongodb-enterprise-tools hold" | sudo dpkg --set-selections
+   service mongod start
+   cd /home/opentech/ && python open-hackathon-server/src/setup_db.py
+}
+
+get_dependency_for_guacamole() {
+    echo "solve dependency software for guacamole"
+    result=$(sudo service guacd restart)
+    if grep -q "SUCCESS" <<< $result; then
+        echo "guacamole is installed!"
+        return
+    fi
+    echo "installing libcairo2-dev"
+    result=$(sudo apt-get install -y libcairo2-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+    echo "installing libjpeg62-dev libpng12-dev libossp-uuid-dev"
+    result=$(sudo apt-get install -y libjpeg62-dev libpng12-dev libossp-uuid-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+    echo "installing libfreerdp-dev libpango1.0-dev libssh2-1-dev libtelnet-dev"
+    result=$(sudo apt-get install -y libfreerdp-dev libpango1.0-dev libssh2-1-dev libtelnet-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+    echo "installing libvncserver-dev libpulse-dev libwebp-dev libssl-dev libvorbis-dev"
+    result=$(sudo apt-get install -y libvncserver-dev libpulse-dev libwebp-dev libssl-dev libvorbis-dev)
+    if grep -q "Unable to lacate" <<< $result; then
+        echo "Could not install dependancy software for guacamole, pls install guacamole manually"
+        exit
+    fi
+}
+
+install_and_config_guacamole() {
+    result=$(sudo service guacd restart)
+    if grep -q "SUCCESS" <<< $result; then
+        echo "guacamole is installed!"
+        return
+    fi
+    echo "installing guacamole"
+    # install and Configure guacamole
+    cd /home/opentech && rm -f download && rm -rf guacamole-server-0.9.9
+    wget http://sourceforge.net/projects/guacamole/files/current/source/guacamole-server-0.9.9.tar.gz/download
+    rm -rf guacamole-server-0.9.9
+    mv download guacamole-server-0.9.9.tar.gz &&  tar -xzf guacamole-server-0.9.9.tar.gz && cd guacamole-server-0.9.9/
+    result=$(autoreconf -fi)
+    result=$(./configure --with-init-dir=/etc/init.d)
+    result=$(make clean)
+    result=$(make)
+    result=$(make install)
+    ldconfig
+    # configure guacamole client
+    wget http://sourceforge.net/projects/guacamole/files/current/binary/guacamole-0.9.9.war/download
+    mv download /var/lib/tomcat7/webapps/guacamole.war
+    # configure guacamole authentication provider
+    mkdir /usr/share/tomcat7/.guacamole
```

Comment:
what's the `owner/group` of this folder? Does user `tomcat7` has enough permission to read/execute it?